xml/schema/decoder-4-15-2.xsd

```diff
@@ -1128,40 +1132,6 @@
     <xs:attribute name="comment" type="xs:string"/>
   </xs:complexType>
 
-  <xs:complexType name="compEnumValType">
```

Comment:
Just checking. Is this element not used any more?

Reply:
Correct. It was a trace from my previous class experiment